capstone/src/main/webapp/script.js

```diff
@@ -326,25 +326,23 @@ function isStringEmpty(str) {
   return (str.length === 0 || !str.trim() || !str);
 }
 
-/** post request params to send a POST request using fetch() */
-const requestParamPOST = {
-  method: 'POST',
-  headers: {
-    'Content-Type': 'application/json',
-  },
-};
-
-function getReviews() {
+async function getReviewsEntities(reviews) {
   // TODO(#33): integrate with actual reviews of businesses
-  const review = 'Really good pizza, nice wine, reasonable prices and great music.';
-  getEntities(review);
+  const reviewsEntities = await getEntities(reviews);
```

Comment:
is there maybe an issue you could attach to here?

Reply:
this is tied w issue#33 (see comment above)

Reply:
ohhh very nice